src/editor/nodes/VideoNode.js

```diff
@@ -62,6 +62,7 @@ export default class VideoNode extends EditorNodeMixin(Video) {
     this._canonicalUrl = "";
     this._autoPlay = true;
     this.volume = 0.5;
+    this.controls = false;
```

Comment:
Right now audio and video nodes will always have controls in hubs: https://github.com/mozilla/hubs/blob/master/src/gltf-component-mappings.js#L192

Not sure if we want to change this behavior. Personally I like being able to change the volume of videos/audio loaded from Spoke (and I like for them to be "focus-able", which right now is tied together), but if we want to have uncontrollable videos/audio nodes we'll have to edit hubs too.